packages/terra-time-input/src/TimeInput.scss

```diff
@@ -46,6 +46,13 @@
         box-shadow: var(--terra-time-input-focus-keyboard-box-shadow-rtl, 0 0 8px #a6d9f4);
       }
     }
+
+    &.disabled {
```

Comment:
Why are we adding diabled styling to the time-input? Can we not use the disabled styles of the input component?

Reply:
For this one, we can not. We have input styles on top of input styles, and the alpha value is causing a highlight effect that we can only remove by making the inputs transparent inside the time input.